tools/pontoon/search_filters.md

```diff
@@ -7,19 +7,15 @@
 ### String status
 
 Strings in Pontoon can be filtered by their status. A string can be in one of the following status:
-* *Missing*: string is not available in the localized file and doesn’t have any translations or suggestions in Pontoon.
-* *Suggested*: at least one suggestion has been submitted for this string. Note that, in this case, the translation only exists within Pontoon database, it’s not saved to the localized file.
-* *Translated*: string has an approved translation.
+* *Missing*: string is not available in the localized file and doesn’t have any approved translations in Pontoon.
 * *Fuzzy*: string is marked as [fuzzy](../../misc/glossary.md#fuzzy) in the localized file.
-
-Suggestions can be in two status:
-* *Unreviewed*: string has been suggested, but not approved or rejected.
+* *Translated*: string has an approved translation.
+* *Unreviewed*: string has been submitted – either to a string with an existing approved translation, or to a missing string – but not reviewed yet by translators. Note that, in both cases, the translation only exists within Pontoon database, it’s not saved to the localized file.
 * *Rejected*: string has been reviewed and rejected by a translator.
```

Comment:
Rejected is covered below, under extra.

I was thinking about adding a new "Review status" section, which would include Unreviewed and Rejected, but then it would also require Translated, which are already included in "Translation status".

Or maybe we should simply move Rejected under Unreviewed (even the latter is not a translation status)?

Reply:
This section is independent from the menu, describing the possible status for a string, and that's why *rejected* is here. Later is mentioned because it's one of the extra filters available. So, it's not really a duplication.

Which makes me think Rejected belongs indeed in the "TRANSLATION STATUS" section. Both "unreviewed" and "rejected" are possible status of a string.

Reply:
Yup, in this case let's removed the 2nd definition of `Rejected`.